Source/DafnyCore/AST/Grammar/TokenNewIndentCollector.cs

```diff
@@ -83,8 +83,9 @@ private class Indentations {
   public int binOpIndent = -1;
   public int binOpArgIndent = -1;
 
-  internal TokenNewIndentCollector(Program program) {
+  internal TokenNewIndentCollector(Program program, Uri uri) {
```

Comment:
What uri is this? My impression is that you want to format one file at a time, which is fine, but let's make that explicit by renaming `uri` to `fileToFormat`